IntegrationTestDS3/IntegrationTestDS3Client.cs

```diff
@@ -79,37 +84,47 @@ public void startup()
        private void setupTestData()
        {
            string root = Path.GetTempPath() + TESTDIR + Path.DirectorySeparatorChar;
-           testdirectorySrc = root + "src" + Path.DirectorySeparatorChar;
-           string testdirectorySrcFolder = root + "src" + Path.DirectorySeparatorChar + FOLDER + Path.DirectorySeparatorChar;
-           testdirectoryDest = root + "dest" + Path.DirectorySeparatorChar;
-           testdirectoryDestPrefix = root + "destPrefix" + Path.DirectorySeparatorChar;
+           testDirectorySrc = root + "src" + Path.DirectorySeparatorChar;
+           string testDirectorySrcFolder = root + "src" + Path.DirectorySeparatorChar + FOLDER + Path.DirectorySeparatorChar;
+           testDirectoryDest = root + "dest" + Path.DirectorySeparatorChar;
+           testDirectoryDestPrefix = root + "destPrefix" + Path.DirectorySeparatorChar;
+
+           testDirectoryBigFolder = root + BIG + Path.DirectorySeparatorChar;
 
-           // create and populate a new test dir
-           if (Directory.Exists(root))
+            // create and populate a new test dir
```

Comment:
This space was fixed for now, but all the file indentation is wrong. we have one of the tabs with only 3 spaces. I'll fix that in a different branch so it will be clear what we changed here for the new test and what is the spaces fix.